xlgui/properties.py

```diff
@@ -80,10 +81,10 @@ def __init__(self, parent, tracks, current_position=0, with_extras=False):
         self.__default_attributes = Pango.AttrList()
         self.__changed_attributes = Pango.AttrList()
 
-        self.message = dialogs.MessageBar(
-            parent=self.builder.get_object('main_container'),
-            buttons=Gtk.ButtonsType.CLOSE
-        )
+        main_container = self.builder.get_object('main_container')
+        self.message = dialogs.MessageBar(buttons=Gtk.ButtonsType.CLOSE)
+        main_container.insert_row(0)
+        main_container.attach(self.message, 0, 0, 6, 1)
```

Comment:
Yeah, I meant a vertical GtkBox.

Regarding Box vs. Grid: We can of course use single-row/single-column Grids (and the docs say they "provide exactly the same functionality"), but I think the Box API is still better for those cases, though I don't know if it's going to be deprecated any time.